### PR TITLE
Update integration page

### LINKED ISSUE #1939: add more ecosystem tools in the integration summary page
### Related code PR

_No response_

### Which part(s) of the docs might be affected or should be updated? And how?

- hasura: A GraphQL gateway. Can be put in the "other tools" section.
- hivemq: A MQTT message queue. Should be in "Message brokers or streaming services". Both source and sink are in progress.
- rabbitmq: Another "Message brokers or streaming services". Researching.
- datadog: A monitoring platform. Suitable to be in the "other tools" section. 
- emqx: A MQTT message queue. Available (https://github.com/risingwavelabs/risingwave/pull/15388).
- neon: Should be considered another Postgres system (https://docs.risingwave.com/docs/current/rw-integration-summary/#postgresql). In progress.
- liquibase: Database management tool. Products like Flyway, Bytebase. DBeaver, PgAdmin and Datagrip are in the same field. Researching.
- Google Pub/Sub: Another "Message brokers or streaming services". Source is available. Sink is in progress.

### Reference

_No response_

### PR DESCRIPTION
**Pull request by WanYixian**
## Info

- **Description**

  - Update integration page, see [preview](https://pr-2073.d2fbku9n2b6wde.amplifyapp.com/docs/dev/rw-integration-summary/).

- **Notes**

  - [ Include any supplementary context or references here. ]

- **Related code PR**

  - [ Provide a link to the relevant code PR here, if applicable. ]

- **Related doc issue**
  
  Resolves https://github.com/risingwavelabs/risingwave-docs/issues/1939

## For reviewers

- **Preview**

  - [ Paste the preview link to the updated page(s) here. Edit this item after the preview site is ready. To find the updated pages, scroll down to locate and open the Amplify preview link and select the **dev** version of the documentation. ]

- **Key points**

  - [ Parts that may need revision or extra consideration. ]

## Before merging

- [ ] I have checked the doc site preview, and the updated parts look good.

- [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`CharlieSYH`, `emile-00`, & `hengm3467`).

### CODE REVIEW

File: docs/rw-integration-summary.md [hunk 1]
```diff
@@ -32,13 +32,17 @@ RisingWave can sink data to Kafka. This allows indirectly sinking data to any sy
...
-| Confluent Cloud | Available. See [Ingest from Confluent Cloud](/guides/confluent-kafka-source.md) for details. | |
 | Amazon MSK | Available. See [Ingest from Amazon MSK](/guides/connector-amazon-msk.md) for details. | |
-|Redpanda | Available. See [Ingest from Redpanda](/ingest/ingest-from-redpanda.md) for details. |Available. See [Sink to Kafka](/guides/create-sink-kafka.md) for details.|
 |Apache Pulsar|Available. See [Ingest from Pulsar](/ingest/ingest-from-pulsar.md) for details. | Available. See [Sink to Pulsar](/guides/sink-to-pulsar.md) for details.|
+| Confluent Cloud | Available. See [Ingest from Confluent Cloud](/guides/confluent-kafka-source.md) for details. | |
 |DataStax Astra Streaming| Available. See [Ingest data from DataStax Astra Streaming](/guides/connector-astra-streaming.md) for details. | Researching <voteNotify note="astra_streaming_sink" />|
-|StreamNative Cloud| Available| Researching <voteNotify note="streamnative_cloud_sink" />|
+|EMQX|Available. See [Ingest from MQTT brokers](/ingest/ingest-from-mqtt.md) for details.| In progress <voteNotify note="emqx" /> |
```
Comment (neverchanje) [inline docs/rw-integration-summary.md:39]: Please refer users to this doc https://docs.risingwave.com/docs/dev/ingest-from-mqtt/

File: docs/rw-integration-summary.md [hunk 2]
```diff
@@ -32,13 +32,17 @@ RisingWave can sink data to Kafka. This allows indirectly sinking data to any sy
...
+|Google Pub/Sub|Available. See [Ingest from Google Pub/Sub](/ingest/ingest-from-google-pubsub.md) for details.| Researching <voteNotify note="google_pubsub_sink" /> |
```
Comment (neverchanje) [inline docs/rw-integration-summary.md]: Pub/sub sink has been In progress. 😄

**Review by neverchanje — APPROVED** [review]
Rest LGTM!

File: docs/rw-integration-summary.md (hunk 2 above)
Comment (WanYixian) [reply docs/rw-integration-summary.md]: Updated now

**Review by hengm3467 — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**